frontend/types/modules.d.ts

```diff
@@ -40,14 +42,25 @@ export interface ModuleWithImages {
   cachedImages: Image[];
 }
 
+export interface Text {
+  text?: string[]; //for quotes.quote
+  description?: string[]; //for sources.source, entities.person
+}
+export interface Meta {
+  score: number;
+  highlight: Text; //array of Text object
+  sort: number[];
+}
+
 export interface Quote extends SearchableModule, ModuleWithImages {
   model: "quotes.quote";
   title: string;
   attributeeHtml: string;
-  attributeeString: string;
+  attributeeString: string; //name of person who is quoted
```

Comment:
IMO we should only put comments describing data model fields in Django's model definitions, but I'll leave that decision to others.